### PR TITLE
Release to bintray

### PR DESCRIPTION
**Pull request by missedone**
For #1

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -21,6 +28,8 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>1.7</maven.compiler.source>
         <maven.compiler.target>1.7</maven.compiler.target>
+        <bintray.repo>test</bintray.repo>
```
Comment (missedone) [inline pom.xml]: we might change this soon

**Comment by missedone** [comment]
for now, as an prototype, the release bits were uploaded to `testng-remote` package in `test` repo:
https://bintray.com/testng-team/test/testng-remote/view#files/org/testng

we need finalize the repo and package name later.

**Comment by missedone** [comment]
ping @cbeust , @juherr

File: pom.xml [hunk 2]
```diff
@@ -86,7 +127,39 @@
...
+                    <artifactId>maven-javadoc-plugin</artifactId>
+                    <version>2.10.3</version>
+                </plugin>
             </plugins>
         </pluginManagement>
+        <plugins>
+            <plugin>
+                <artifactId>maven-release-plugin</artifactId>
+                <configuration>
+                    <useReleaseProfile>false</useReleaseProfile>
```
Comment (juherr) [inline pom.xml]: For my information, can you tell me why you set it to false (instead of using the `release-profile`)?

**Comment by juherr** [comment]
:+1:

File: pom.xml (hunk 2 above)
Comment (missedone) [inline pom.xml]: right, i don't see any reason need this.
it was copied from here: http://veithen.github.io/2013/05/26/github-bintray-maven-release-plugin.html

now removed.

Comment (juherr) [inline pom.xml]: Maybe because the default release plugin is trying to deploy the site too?
Just try another release before the merge ;)

**Comment by missedone** [comment]
just tested, it works even without maven-release-plugin configuration: https://bintray.com/testng-team/test/testng-remote/1.0.1/view#files/org/testng

**Comment by juherr** [comment]
Awesome! :+1: 

Do you plan to remove commits from the release test?

**Comment by missedone** [comment]
yes, i reverted the git commit (and tag as well), will remove the `test` binary repo later.

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+1. Add new Version for the repo package on Bintray
+  Follow the doc to create Version: https://bintray.com/docs/usermanual/working/working_versions.html 
+  for example, to release `1.0.0` 
+
+### Perform release
+
+One the release server, run the following maven command to release (tagging and upload to Bintray):
+```bash
+REL_VER=1.0.0
+DEV_VER=1.0.0-SNAPSHOT
```
Comment (juherr) [inline RELEASE.md]: > developmentVersion: Default version to use for new local working copy.

@missedone Could you confirm the `1.0.0-SNAPSHOT` value? How is the next dev version chosen?

Comment (missedone) [inline RELEASE.md]: it's typo, should be `1.0.1-SNAPSHOT`

**Comment by missedone** [comment]
typo fixed, please merge if you find no issue:)
thanks

**Pull request merged**